### PR TITLE
FIXED: minimumAvailability spec for lists missing(was in develop)

### PR DESCRIPTION
**Pull request by geogolem**
#### Database Migration
NO

#### Description

minimumavailability specification for items added via netimport lists

#### Todos
- [ ] Tests

#### Issues Fixed or Closed by this PR
this was in develop but didn't get ported over to aphrodite. 
* #

### CODE REVIEW

**Comment by geogolem** [comment]
@Qstick 
@onedr0p 

Hopefully this is ok. I think I know how to deal with things in my environment. I started running the entire js files through the linter in vscode, but I realize I can't really do that.  I just created this new PR in place of the old one.

**Comment by Qstick** [comment]
Looks good 👍

File: frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ import FormLabel from 'Components/Form/FormLabel';
 import FormInputGroup from 'Components/Form/FormInputGroup';
 import ProviderFieldFormGroup from 'Components/Form/ProviderFieldFormGroup';
 import styles from './EditNetImportModalContent.css';
+import availabilityOptions from 'Components/Form/AvailabilitySelectInput';
```
Comment (Qstick) [inline frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js]: You don't need this when specifically using the input type for availability select

File: frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js [hunk 2]
```diff
@@ -115,6 +117,17 @@ function EditNetImportModalContent(props) {
                 />
               </FormGroup>
 
+              <FormGroup>
+                <FormLabel>Minimum Availability</FormLabel>
+                <FormInputGroup
+                  type={inputTypes.AVAILABILITY_SELECT}
+                  name="minimumAvailability"
+                  values={availabilityOptions}
```
Comment (Qstick) [inline frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js]: No need to pass in values, it's automatic in the availability select input type

**Comment by Qstick** [comment]
Code looks good to me now, have not had a chance to actually test it though. Will run it tonight and merge

**Pull request merged**